src/Clipper.jl

```diff
@@ -991,16 +1026,28 @@ function Base.show(io::IO, p::__ClipperPath)
     if isempty(p)
         return
     end
+    print(io, "Path([")
     for i = 1:length(p)-1
         print(io, string("(", x(p[i]), ",", y(p[i]), "), "))
     end
     print(io, string("(", x(p[end]), ",", y(p[end]), ")"))
+    print(io, "])")
 end
 
 function Base.show(io::IO, p::__ClipperPaths)
-    for i = 1:length(p)
-        show(io, p[i])
+    len = length(p)
+    if len == 0
+        print(io, "Paths()")
+        return
+    end
+    print(io, "Paths([\n    ")
+    for i = 1:len
+        print(io, p[i])
+        if i < len
+            print(io, ",\n    ")
```

Comment:
Julia's arrays use newlines in their output:

```
julia> a = [1 2;3 4]
2x2 Array{Int64,2}:
 1  2
 3  4
```

I think newlines make the arrays significantly easier to read, and I think the same can be said about `Paths()`